packages/backend/src/build-disk-image.ts

```diff
@@ -44,7 +44,7 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
 
   // VALIDATION CHECKS
   for (const { field, message } of requiredFields) {
-    if (!build[field]) {
+    if (!build[field as keyof BootcBuildInfo]) {
```

Comment:
for this one it's look tricky

Reply:
Yeah. I like that it actually says what it's doing though, and other options I tried were more verbose, e.g.:
`if (!(build as {[key: string]: any})[field]) {`
or:
`if (!(build as Record<string, any>)[field]) {`
or:
```
const obj: Record<string, any> = build;
if (!obj[field]) {
```

Prefer any of these or a different one?

Reply:
Merging, feel free to ping if you'd like me to change it :).

Reply:
ah yes no pb, I was hoping that we don't need to use any cast there but maybe it's not possible but anyway it's was not preventing any merge 👍